mne/utils/docs.py

```diff
@@ -874,8 +874,28 @@
 docdict['rank'] = """
 rank : None | dict | 'info' | 'full'
     This controls the rank computation that can be read from the
-    measurement info or estimated from the data. See ``Notes``
-    of :func:`mne.compute_rank` for details."""
+    measurement info or estimated from the data.
+
+    .. note:: For Maxwell-filtered data, you will typically want set this to
+              ``'info'``.
```

Comment:
I would say this is actually not true if you use movement compensation

Reply:
Any suggestion on how to phrase this better?

Background is that empirical rank determination doesn't work properly for @SophieHerbst with the default settings: rank is always equal to number of channels. Only with `rank='info'` do we get what looks like a more "trustworthy" rank. We figured that we somehow should inform users about this. Maybe we can take an entirely different approach in communication here? I know too little about this stuff to come up with really useful proposals, I'm afraid! (as just demonstrated!)

Reply:
It might depend on whether Python or Elekta MaxFilter was used (?) but in general what I do is do everything in Python, compute rank on epochs, then pass this as the `rank` param in all functions (like cov and inverse computation, etc.). I don't usually have trouble with rank when doing this, so we should figure out what the problem is.

For now I would just remove this extra bit of wording (it's an orthogonal issue) and discuss in https://github.com/mne-tools/mne-python/issues/4676 what the right thing is to do, probably by sharing files and figuring out what the problems actually are.